pkg/ddevapp/traefik_dynamic_config_example_template.yaml

```diff
@@ -0,0 +1,29 @@
+#ddev-generated
+http:
+  routers:
+    {{ .App.Name }}-web-80-http:
+      middlewares:
+        - {{ .App.Name }}-redirectHttps
+        - {{ .App.Name }}-my-fail2ban
+    {{ .App.Name }}-web-80-https:
+      middlewares:
+        - {{ .App.Name }}-my-fail2ban
+
+  middlewares:
+    {{ .App.Name }}-redirectHttps:
+      redirectScheme:
+        scheme: https
+        permanent: true
+    {{ .App.Name }}-my-fail2ban:
+      plugin:
+        fail2ban:
```

Comment:
I think this template should have a comment that you need to enable fail2ban with `static_config.plugins.yaml` or it won't work.

Reply:
Yes, that's correct, but people often don't read the docs or don't read them carefully enough.

It looks fine to me, I'm just wondering how it would look to others.